templates/components/brand-bar/brand-bar.html.twig

```diff
@@ -0,0 +1,20 @@
+{#
+/**
+ * @file
+ * Brand Bar Component.
+ *
+ * Stanford brand bar with the wordmark logo with four variants:
+ *   - Default: White wordmark over cardinal red background
+ *   - Bright: White wordmark over bright red background
+ *   - Dark: White wordmark over dark grey background
+ *   - White: Cardinal red wordmark over white background
+ *
+ * Available variables:
+ * - attributes: For additional HTML attributes not already provided.
+ * - modifier_class: Additional css classes to change look and behaviour.
+ */
+#}
+
+<div class="su-brand-bar su-brand-bar--dark {{ modifier_class }}">
```

Comment:
I'm seeing an additional unnecessary `su-brand-bar--` added to the html when I inspect. I think if you remove `{{ modifier_class }}` that should fix that.
![Screen Shot 2022-01-27 at 4 19 52 PM](https://user-images.githubusercontent.com/49299083/151465465-d2e6a183-fb71-4e61-be90-17940405d35c.png)

Somewhat related brand bar/global footer question - do we want those two components to have the same height?